vue-app/src/components/nav-bar.vue

```diff
@@ -49,9 +49,11 @@ export default {
       <p class="menu-label">Auth</p>
       <div class="menu-list auth">
         <template v-if="!userInfo">
-          <template v-for="provider in providers">
-            <AuthLogin :key="provider" :provider="provider" />
-          </template>
+          <AuthLogin
```

Comment:
removed due to lint rule in Vue, i.e said it doesn't need to be wrapped in a template like that